bounce/server/resource/__init__.py

```diff
@@ -70,16 +70,42 @@ def validate(request_cls, response_cls):
             body should match
     """
 
+    def set_defaults(schema, info, parent_key=None):
+        # if there is no value specified for the keys
+        # set the default values if there are
+        # corresponding default values found in the json schema
+        import pdb
+        pdb.set_trace()
+        for key in schema:
+            value = schema[key]
+            if key == 'default':
+                if parent_key not in info:
+                    info.update({parent_key: value})
+                elif not info[parent_key]:
+                    info.update({parent_key: value})
```

Comment:
Minor: I think you can condense these two `if` statements into 

```python
if parent_key not in info or not info[parent_key]:
```

Reply:
got it.